File: Latihan B/Python/Hardware.py
from Product import Product

#kelas yang digunakan untuk mengimplementasikan kelas Hardware
class Hardware(Product):

	#konstruktor
	def __init__(self):
		self.brand = ""
		self.model = ""

	#getter dan setter untuk atribut brand
	def setBrand(self, brand):
		self.brand = brand

	def getBrand(self):
		return self.brand

	#getter dan setter untuk atribut model
	def setModel(self, model):
		self.model = model

	def getModel(self):
		return self.model

File: Latihan B/C++/Hardware.cpp
//kelas yang digunakan untuk mengimplementasikan kelas Hardware
class Hardware : public Product{

	private:

		//atribut private
		string brand;
		string model;

	public:

		//konstruktor
		Hardware(string brand, string model){
			this->brand = brand;
			this->model = model;
		}

		Hardware(){
			this->brand = "";
			this->model = "";
		}

		//getter dan setter brand
		void setBrand(string brand){
			this->brand = brand;
		}

		string getBrand(){
			return this->brand;
		}

		//getter dan setter model
		void setModel(string model){
			this->model = model;
		}

		string getModel(){
			return this->model;
		}

		//destruktor
		~Hardware(){

		}

};